scripts/postinstall.js

```diff
@@ -84,6 +84,15 @@ const run = async () => {
       },
     ])
   );
+  //ToDo: Remove when opensearch-js is released to include https://github.com/opensearch-project/opensearch-js/pull/889
+  promises.push(
+    patchFile('node_modules/@opensearch-project/opensearch-next/lib/Serializer.js', [
+      {
+        from: 'val < Number.MAX_SAFE_INTEGER',
+        to: 'val < Number.MIN_SAFE_INTEGER',
```

Comment:
seems js client had a release, is this still needed?

Reply:
I guess we can raise another PR if it's not needed?

Reply:
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/8886 bump to 2.13.0 marked with label `3.0.0`, we might need to keep it for 2.x

Reply:
2.x uses the JS client with a `-next` suffix so the change has to be manually ported to 2.x.